drivers/i2c/i2c_npcx_controller.c

```diff
@@ -820,6 +863,87 @@ int npcx_i2c_ctrl_get_speed(const struct device *i2c_dev, uint32_t *speed)
 	return 0;
 }
 
+int npcx_i2c_ctrl_recover_bus(const struct device *dev)
+{
+	struct smb_reg *const inst = HAL_I2C_INSTANCE(dev);
+	int ret = 0;
+
+	i2c_ctrl_bank_sel(dev, NPCX_I2C_BANK_NORMAL);
+
+	/* Enable writing to SCL_LVL/SDA_LVL bit in SMBnCTL3. */
+	inst->SMBCTL4 |= BIT(NPCX_SMBCTL4_LVL_WE);
```

Comment:
Is this enable needed?  I see that all the helper functions that write to SMBnCTL3 already set and clear this bit.

Reply:
You are right. This is not needed. I have removed it in the new patch set.